docs/Makefile

```diff
@@ -15,3 +15,6 @@ artwork:
 
 clean:
 	rm book.pdf ch*.html cover-front-web.png
+
+spell:
+	for f in ../tex/*.tex; do cat $$f | aspell -t list; done | sort | uniq | comm -2 -3 - ../tex/sorted_words_to_ignore.txt
```

Comment:
Please consider making `aspell` invocation locale-independent.
For example, on my machine `make spell` results in
```
Error: No word lists can be found for the language "ru_RU".
```

Setting the `--master` option to `en_US` fixes it for me:
```bash
aspell --master=en_US -t list
```